tests/test_waypoints_to_spline_commands.py

```diff
@@ -15,8 +15,10 @@ def test_waypoints_to_spline_commands_empty_input():
     waypoints = []
     altitude = 100
 
-    result, commands_actual = waypoints_to_spline_commands.waypoints_to_spline_commands(waypoints, 
-                                                                                        altitude)
+    result, commands_actual = waypoints_to_spline_commands.waypoints_to_spline_commands(
+                            waypoints, 
+                            altitude
```

Comment:
This should be intended by 1 more than `result, ... =` . See `test_waypoints_names_to_coordinates.py` for an example.